tests/test_urls.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# Download and build the data if it does not exist.
+
+import unittest
+import importlib
+import os
+import warnings
+import sys
+
+TO_SKIP = [
+    './parlai/tasks/__pycache__',
+    './parlai/tasks/interactive',
+    './parlai/tasks/fromfile',
+    './parlai/tasks/taskntalk',
+    './parlai/tasks/integration_tests',
+    './parlai/tasks/dialog_babi_plus',
+    './parlai/tasks/decanlp',
+]
+
+SPECIFIC_BUILDS = {
+    './parlai/tasks/opensubtitles': ['build_2009', 'build_2018'],
+    './parlai/tasks/coco_caption': ['build_2014', 'build_2015', 'build_2017'],
+}
+
+
+class TestUtils(unittest.TestCase):
+    def test_http_response(self):
+        sys.path.insert(0, './lib')
+        tasks = [f.path for f in os.scandir('./parlai/tasks') if f.is_dir()]
```

Comment:
so instead I'd recommend

```python
import parlai.tasks
for task_module in dir(tasks):
```

or similar. I'd prefer to inspect the code.

Reply:
I'm not completely sure how I could utilize this.
```
import parlai.tasks as tasks

for task_module in dir(tasks):
```
only has the following attributes
```
__builtins__
__cached__
__doc__
__file__
__loader__
__name__
__package__
__path__
__spec__
```

Should it enumerate all tasks in the `__init__.py` file?
Also, would this change make the test to be 'nightly'?